src/components/routes/CalendarItemDisplay.vue

```diff
@@ -0,0 +1,241 @@
+            />
+          </svg>
+        </q-icon>
+      </template>
+
+      <!-- Content: Route logged -->
+      <template v-if="route && route.id">
+        <!-- Icon: Transport type -->
+        <q-icon
+          :color="active ? 'white' : 'primary'"
+          :name="getRouteIcon(route.transport)"
+          size="18px"
```

Comment:
size attr value 18px should be const (repeat > 1).

Reply:
Updated.